packages/renderer/src/lib/deployments/DeploymentColumnConditions.svelte

```diff
@@ -9,33 +18,74 @@ export let object: DeploymentUI;
 
 // Determine both the icon and color based on the deployment condition
 function getConditionAttributes(condition: DeploymentCondition) {
-  let name = condition.type;
-  switch (condition.type) {
-    case 'Available':
-      // faCheckCircle: Indicates a successful state, typically used to denote availability and operational readiness
-      return { color: 'text-green-600', icon: faCheckCircle, name };
-    case 'Progressing':
-      // faSync: Often used to represent ongoing processes or operations, fitting for a "Progressing" state
-      // If reason has NewReplicaSetAvailable then it's progressed
-      if (condition.reason === 'NewReplicaSetAvailable') {
-        name = 'Progressed';
-      }
+  const defaults = {
+    name: condition.type,
+    color: 'text-gray-500',
+    icon: faQuestionCircle,
+  };
 
-      return { color: 'text-sky-400', icon: faSync, name };
-    case 'ReplicaFailure':
-      // faExclamationTriangle: Alerts and warnings
-      return { color: 'text-amber-600', icon: faExclamationTriangle, name };
-    default:
-      // faQuestionCircle: Uncertain / unknown
-      return { color: 'text-gray-900', icon: faQuestionCircle, name };
-  }
+  // Condition map for easier lookup
+  const conditionMap: { [key: string]: { name: string; color: string; icon?: IconDefinition } } = {
+    'Available:MinimumReplicasAvailable': {
+      name: 'Available',
+      color: 'text-[var(--pd-status-running)]',
```

Comment:
Maybe easier to read/less text if this just returns the color variable and outer code adds the rest?